src/Spinner.elm

```diff
@@ -135,44 +145,46 @@ defaultConfig =
     , translateY = 50
     , shadow = True
     , hwaccel = False
-    , color = always white
+    , color = always "rgba(255, 255, 255, 0.9)"
```

Comment:
As mentioned, I only did this so that I could verify that the package compiles in Elm 0.19.